cypress/integration/focus-trap-demo.spec.js

```diff
@@ -17,6 +17,25 @@ describe('focus-trap', () => {
     cy.get(focusedElSelectorOrAliasInTrap).should('be.focused');
   }
 
+  /**
+   * Verify focus trap is **NOT** trapping focus by tab
+   * @param cyWrapper Cypress object of outside focused element yielded from `cy.get/contains/findByRole...etc`
+   */
+  function verifyFocusIsNotTrapped(cyWrapper) {
+    const outsideFocusedElAlias = `outsideFocusedEl`;
```

Comment:
🤔  nit: I'm surprised ESLint didn't complain about backticks for a string that doesn't use variable references. Must be a rule I expect is "recommended" but isn't.